runtime/compiler/z/codegen/J9CodeGenerator.cpp

```diff
@@ -4005,6 +4012,14 @@ J9::Z::CodeGenerator::inlineDirectCall(
             return resultReg != NULL;
             }
       break;
+
+      case TR::java_lang_StringCoding_hasNegatives:
+         if (cg->getSupportsInlineStringCodingHasNegatives())
+            {
+            resultReg = TR::TreeEvaluator::inlineStringCodingHasNegatives(node, cg);
+           return resultReg != NULL;
```

Comment:
I understand that other code before this checks resultReg, but looking at the tree evaluator, are you ever going to have NULL resultReg? May be simply return `true` here?

Reply:
Returning `true` unconditionally now: 7f22c4fb94e